### ISSUE TITLE
最新fluid api如何实现多任务

### ISSUE THREAD

**Issue by kjfren1985** [PaddlePaddle/Paddle#15564]
如题，现在想用paddle做multi task训练，例如同时拟合用户的性别及婚姻状况，请问在最新的fluid api中如何实现？
issue中找到了两个相关的讨论：
https://github.com/PaddlePaddle/Paddle/issues/7511
https://github.com/PaddlePaddle/Paddle/issues/4191
但是貌似中间提到的写法和现在不太一样了（比如类似extra_layers = [loss1, loss2]？），请问最新版本中如何实现？

**Comment by dzhwinter** [comment]
多任务训练写多个loss求和就可以，你提到的两个case是老API的，现在写法有一点变化。
loss1 = fluid.layers.xxx_loss()
loss2 = fluid.layers.xxx_loss2()
loss = loss1 + loss2
opt = fluid.optimizer.SGD()
opt.minimize(loss)

**Comment by dzhwinter** [comment]
这样能否满足你的需求？

**Comment by kjfren1985** [comment]
多谢，我尝试下~

**Comment by dkerjw1** [comment]
> 多任务训练写多个loss求和就可以，你提到的两个case是老API的，现在写法有一点变化。
> loss1 = fluid.layers.xxx_loss()
> loss2 = fluid.layers.xxx_loss2()
> loss = loss1 + loss2
> opt = fluid.optimizer.SGD()
> opt.minimize(loss)

@dzhwinter 你好，请问如果我有多个输出层，分别输出并计算loss，如何让每个loss更新对应的输出层的参数呢？
